bin/run_docker.py

```diff
@@ -42,6 +42,51 @@ def get_submission_image(syn: synapseclient.Synapse, submission_id: str) -> str:
     return image_id
 
 
+def make_invalid_output(file_name, log_file_path, file_content):
+
+    dummy_file_name = "INVALID_"+file_name
+    dummy_file = os.path.join(log_file_path, dummy_file_name)
+    with open(dummy_file, "w") as d:
+        d.write(file_content)
+
+    return dummy_file
+
+
+def handle_outputs(output_path: str, output_file_name: str, log_text: str):
+    """
+    Handles any output files generated by the submitted Docker container:
+    Checks if any output files were generated by the container
+    run. If any were found, it will return the path to the first one found.
+    If none were found, it will return None.
+
+    Arguments:
+        output_path: The path to the output directory.
+        output_file_name: The name of the output file.
+        log_text: The log text to be updated, if an incorrect number of
+                  output files were found.
+
+    Returns:
+        output_file: The path to the output file.
+        log_text: The log text that was updated, if an incorrect number of
+                  output files were found.
+
+    """
+    # Glob any expected output files stored in the output_dir as a result of the container run
+    file_glob = glob(os.path.join(output_path, output_file_name+".*"))
+
+    # Update the log file if there is a case where too many output files were generated, or none were generated
+    if len(file_glob) != 1:
+        no_output_msg = f"Expected 1 Docker container output file in the output directory. Got {len(file_glob)}. If multiple output files were generated, the first one retrieved will be used for validation and scoring."
+        if isinstance(log_text, bytes):
+            log_text = log_text.decode("utf-8")
+        log_text = log_text + "\n" + no_output_msg
```

Comment:
With the "dummy predictions file" method, `UPDATE_FOLDERS` can still be part of the workflow, and so I decided to go back to updating the Docker exec logs with the unexpected output file msg and I'm liking how this is turning out: 
<img width="774" alt="image" src="https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/assets/32107699/af1ef931-11a3-4483-b972-de98f497428e">